openfisca_canada/variables/canadian_emergency_relief_benefit.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+# This file defines variables for the modelled legislation.
+# A variable is a property of an Entity such as a Person, a Household…
+# See https://openfisca.org/doc/key-concepts/variables.html
+
+# Import from openfisca-core the common Python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the Entities specifically defined for this tax and benefit system
+from openfisca_canada.entities import *
+
+class is_a_resident_of_canada(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = ETERNITY
+    label = 'Asks whether the applicant is a resident of Canada.'
+
+
+class  believe_they_cannot_get_EI(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = ETERNITY
+    label = 'Asks whether the applicant believes they cannot recieve' \
+            'EI Regular or Sickness Benefits'
+
+class has_not_quit_their_job_voluntarily(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = ETERNITY
+    label = 'Asks whether the applicant did not quit their job'
+
+class will_lose_income_for_half_of_the_four_week_period(Variable):
```

Comment:
The tense in the variable does not match the tense in the label.

Reply:
updated they now match